.github/workflows/application.yml

```
@@ -24,22 +24,15 @@ jobs:
     static-checks:
         runs-on: ubuntu-latest
 
-        name: "Static checks (PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}), API Platform ${{ matrix.api-platform }})"
+        name: "Static checks (PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }})"
 
         timeout-minutes: 10
 
         strategy:
             fail-fast: false
             matrix:
                 php: ["8.0", "8.1"]
-                symfony: ["^5.4"]
-                api-platform: ["^2.7"]
-
-                include:
-                    -
-                        php: "8.1"
-                        symfony: "^5.4"
-                        api-platform: "~2.6.0"
+                symfony: ["^5.4", "^6.0"]
```

Shouldn't we test Symfony 6.0 and 6.1, or only the newest one?
```suggestion
                symfony: ["^5.4", "~6.0.0", "~6.1.0"]
```
Or maybe at least include one build for 6.0 🤔 
Of course here and in other workflows 😄